AnimateSvg.js

```diff
@@ -108,7 +120,7 @@ export default function AnimateSvg(props) {
       height: '100%',
       width: '100%',
     }}>
-      {props.children}
+      <LogoSvg />
```

Comment:
LogoSvg being returned within AnimateSvg component return statement as opposed to previously being returned within top level App component return statement which was originally using props.children

Reply:
rewrite this to be more clear

Reply:
How about: Changed this from a `children` pattern to directly passing in `LogoSvg` to have easier access to the props in `Animated.View`